src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/OpprettRevurderingBehandlingService.kt

```diff
@@ -73,38 +71,45 @@ class OpprettRevurderingBehandlingService(
         return behandling.id
     }
 
-    private fun validerValgteBarn(request: OpprettBehandlingDto, forrigeBehandlingId: UUID) {
+    private fun validerValgteBarn(request: OpprettBehandlingDto, behandlingIdForGjenbruk: UUID?) {
         val stønadstype: Stønadstype by lazy { fagsakService.hentFagsak(request.fagsakId).stønadstype }
         feilHvis(request.valgteBarn.isNotEmpty() && stønadstype != Stønadstype.BARNETILSYN) {
             "Kan ikke sende inn barn til $stønadstype"
         }
+        if (stønadstype != Stønadstype.BARNETILSYN) {
+            return
+        }
 
         feilHvis(!request.årsak.erSøknadEllerPapirsøknad() && request.valgteBarn.isNotEmpty()) {
             "Kan ikke sende med barn på annet enn årsak Søknad"
         }
 
-        val barnTilRevurdering = hentBarnTilRevurdering(request.fagsakId, forrigeBehandlingId).barn
+        val barnTilRevurdering = hentBarnTilRevurdering(request.fagsakId, behandlingIdForGjenbruk).barn
 
         val valgbareIdenter = barnTilRevurdering.filterNot { it.finnesPåForrigeBehandling }.map { it.ident }
         feilHvis(!valgbareIdenter.containsAll(request.valgteBarn)) {
             "Kan ikke velge barn som ikke er valgbare."
         }
+
+        feilHvis(behandlingIdForGjenbruk == null && request.valgteBarn.isEmpty()) {
+            "Behandling må opprettes med minimum 1 barn"
```

Comment:
Ikke helt koblet til denne PR egentlige.
Risikerer vel kanskje at de velger 1 barn bare fordi de må, selv om de egentlige ikke har noe barn de egentlige ønsker å opprette behandling for